packages/client/lib/miner/pendingBlock.ts

```diff
@@ -272,13 +317,13 @@ export class PendingBlock {
    * @param blockHash the blockhash of the pending block (computed from the header data provided)
    */
   private constructBlobsBundle = (
-    payloadId: Buffer,
+    payloadId: string,
```

Comment:
These payloadIds are 8-byte strings, right? Should we change the string type to a `buffer8string` or something similar for clarity?

Reply:
so just have a `type buffer8string= string`?